airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -377,11 +377,13 @@ def _infer_field_types_from_df(df: pandas.DataFrame) -> Dict[Any, Any]:
                     filepath=f.name, table=table, delimiter=delimiter, field_dict=field_dict, **kwargs
                 )
 
+    # pylint: disable=too-many-arguments
     def load_file(
         self,
         filepath: str,
         table: str,
         delimiter: str = ",",
+        quotechar: str = '\\"',
```

Comment:
This should just be

```suggestion
        quotechar: str = '"',
```

and then any escaping necessary should be handled in the SQL generation.